src/analysis/complex/conformal.lean

```diff
@@ -0,0 +1,181 @@
+/-
+Copyright (c) 2021 Yourong Zang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yourong Zang
+-/
+import analysis.complex.isometry
+import analysis.calculus.conformal
+
+/-!
+# Complex Conformal Maps
+
+We prove the sufficient and necessary conditions for a complex function to be conformal.
+
+## Main results
+
+* `is_complex_or_conj_complex_linear_iff_is_conformal_map`: the main theorem for linear maps.
+* `conformal_at_iff_holomorphic_or_antiholomorphic_at`: the main theorem for complex functions.
+
+## Warning
+
+Antiholomorphic functions such as the complex conjugate are considered as conformal functions in
+this file.
+-/
+
+noncomputable theory
+
+open complex linear_isometry linear_isometry_equiv continuous_linear_map
+     finite_dimensional linear_map
+
+section complex_normed_space
+
+variables {E : Type*} [normed_group E] [normed_space ℝ E] [normed_space ℂ E]
+  [is_scalar_tower ℝ ℂ E] {z : ℂ} {g : ℂ →L[ℝ] E} {f : ℂ → E}
+
+lemma differentiable_at_iff_exists_linear_map (hf : differentiable_at ℝ f z) :
+  differentiable_at ℂ f z ↔ ∃ (g' : ℂ →L[ℂ] E), g'.restrict_scalars ℝ = fderiv ℝ f z :=
```

Comment:
Can you explain the situation with these sorries?  Is it just that you haven't merged in the changes from the other PR?

Reply:
Yes the sorries are just lemmas from #8418. Once that PR is passed I will remove the worries.

Reply:
Can you please merge that branch into this one?  It is a totally normal process, when you have a PR with dependencies, to keep it updated when there are major changes to the dependencies.

Reply:
Is there a git command I can use to merge that branch? I have no clue on the proper way to do this.

Reply:
Approximately,
```
git checkout new-complex-conformal-branch
git pull
git checkout complex-conformal-functions
git merge new-complex-conformal-branch
[possibly some merge steps]
git push
```
and then wait for Github to compile the new version, then pull and make any adjustments to `complex-conformal-functions` to work with the changes in `new-complex-conformal-branch`.

The first couple times it's easier with help; see if someone on Zulip new members is free to help you on a video call.

Reply:
Thank you!